src/alacris_lib/side_chain_null_operation.ml

```diff
@@ -0,0 +1,50 @@
+open Legilogic_lib
+open Action
+open Signing
+
+open Legilogic_ethereum
+open Side_chain_server_config
+open Side_chain
+
+let side_chain_null_operation_log = true
```

Comment:
I removed the side_chain_null_operation and actually all the null_operation from the file.
I think it is needed to have this as an option since we need it only for tests, not for operation on a normal test net